tools/lxdclient/remote.go

```diff
@@ -44,8 +44,16 @@ var CloudImagesRemote = Remote{
 	ServerPEMCert: "",
 }
 
+var CloudImagesDailyRemote = Remote{
```

Comment:
Pre-existing code, but not of the exported consts or vars etc in this file are commented and they should be.
Would be great to see a driveby here to fix that